homeassistant/components/overkiz/const.py

```diff
@@ -98,6 +98,7 @@
     UIWidget.ATLANTIC_PASS_APC_HEATING_AND_COOLING_ZONE: Platform.CLIMATE,  # widgetName, uiClass is HeatingSystem (not supported)
     UIWidget.ATLANTIC_PASS_APC_HEATING_ZONE: Platform.CLIMATE,  # widgetName, uiClass is HeatingSystem (not supported)
     UIWidget.ATLANTIC_PASS_APC_ZONE_CONTROL: Platform.CLIMATE,  # widgetName, uiClass is HeatingSystem (not supported)
+    UIWidget.ATLANTIC_PASS_APC_HEAT_PUMP: Platform.CLIMATE,  # Used for turning on/off heat pump and selecting cool or heat mode (not for adjusting temperature)
```

Comment:
```suggestion
    UIWidget.ATLANTIC_PASS_APC_HEAT_PUMP: Platform.CLIMATE,   # widgetName, uiClass is HeatingSystem (not supported)
```

Please use the same style as the other line items. We leverage the UIWidget as an override, thus good to add which UIClass your device normally has.

Reply:
Can you move it up as well in the list? We sort this list alphabetically.